src/hb_opts.erl

```diff
@@ -362,7 +362,10 @@ default_message() ->
             [
                 <<"fcoN_xJeisVsPXA-trzVAuIiqO3ydLQxM-L4XbrQKzY">>,
                 <<"WjnS-s03HWsDSdMnyTdzB1eHZB2QheUWP_FVRVYxkXk">>
-            ]
+            ],
```

Comment:
No need for the defaults here I think, as they are already in the `hb_opts:get` calls?